apps/snitch_core/lib/core/data/schema/payment/card_payment.ex

```diff
@@ -22,13 +22,14 @@ defmodule Snitch.Data.Schema.CardPayment do
     field(:cvv_response, :string)
 
     belongs_to(:payment, Payment)
+    belongs_to(:card, Card)
 
     timestamps()
   end
 
   @update_fields ~w(response_code response_message avs_response cvv_response)a
   @create_fields [:payment_id | @update_fields]
-
+  @temp ~w(card_id)a
```

Comment:
I think `card_id` should be part of `@update_fields`

Reply:
yes it can be.